Everest/Everest/Models/Action.swift

```diff
@@ -13,23 +13,20 @@ import SwiftyJSON
 class Action: NSObject {
   
   var id: String          //same id from acts
-  var categoryId: String //same categoryId Id from acts
   var createdAt: String
```

Comment:
Why is this removed?

Reply:
Categories won't have an id, their title will server as an identifier as well (since they are limited in number and their titles are small in size). This reduction was needed for auto filling for the actPicker list through json import. We can add it later if needed through manual edit on FB.

Reply:
category will be a string like "surprise". its more like a type not id.

Reply:
I see what you see, but it can work as an id as well for the reason mentioned above.

Reply:
im ok with id OR type. my point is that there should be a reference to the type of category from a given act.
we will also need this when we support user created acts.

Reply:
also we need to add a field like "type" whose value can be "system or user"

Reply:
We can disallow users from adding via the app. But when we do support adding it will be good to have the context of which system created it.

Reply:
It doesn't harm to add it if you want.